### PR TITLE
improving README file

### PR DESCRIPTION
**Pull request by magusd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,30 +3,30 @@
 
 <p align="left" >
 <img src="https://s-media-cache-ak0.pinimg.com/736x/aa/bc/3b/aabc3b2b789f478ffb87ac2f0bdd2d33--ergo-proxy-manga-anime.jpg" width="250" align="center" />
-<span>Ergo Proxy - The reverse proxy agent for local domain management.</span>
+<span>Ergo Proxy - Reverse proxy agent for local domain.</span>
```
Comment (cristianoliveira) [inline README.md]: I am not sure about this. it sounds strange to me. I would rather stick with the previous version cause it explains better the app purpose. What do you think?

File: README.md [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
 **Feedback**
 
-This project is under development but it's ready to use. Feel free to give me
+This project is still under development but it's ready to use. Feel free to leave your 
 feedback and opening issues. Suggestions and contributions are welcome. :)
 
 ## Why?
 
-Dealing with multiple apps locally is really annoying. Having to remember each port that represents each service, and this gets even worse when you have microservices. So I wanted a simple way to give each app a proper local domain. Ergos comes to solve this simple problem.
+When dealing with multiple apps locally it's really annoying rememberinf which port represents each service and it gets even worse when you have microservices. Ergo solves this by assigning a local domain for each app.
```
Comment (cristianoliveira) [inline README.md]: - I would stick with just `Dealing with multiple...`.
- There is a typo here `rememberinf which` :)

The rest makes sense to me. It makes more impersonal.

File: README.md [hunk 3]
```diff
@@ -45,7 +45,8 @@ curl -s https://raw.githubusercontent.com/cristianoliveira/ergo/master/install.s
 For windows you can find the executable in each [release](https://github.com/cristianoliveira/ergo/releases).
 
 ***Disclaimer:***
-I only use unix based systems on a daily basis, so I am not able to test each build :(
+
+I use unix based systems on a daily basis, so I can't test every build on every OS :(
```
Comment (cristianoliveira) [inline README.md]: Actually, this is more related to Windows-based systems hehe

**Comment by cristianoliveira** [comment]
Referst to #37

File: README.md (hunk 3 above)
Comment (magusd) [reply README.md]: Hahaha I saw that. Didn't want to make it so obvious like "I'm not testing exe, good luck"

Comment (cristianoliveira) [reply README.md]: Ahhhhh I got ya! Good idea :)

**Comment by cristianoliveira** [comment]
Hello @magusd you still have interest in this change?

**Comment by cristianoliveira** [comment]
Close due not answer.

**Pull request closed without merging**